### ISSUE TITLE
MySQL syntax error due to lack of support for 'drop user if exists' on MySQL 5.5

### ISSUE THREAD

**Issue by annabel82** [failys/cairis#189]
### Expected behaviour
setupDb.sh completes successfully.

### Actual behaviour
MySQL syntax error due to lack of support for 'drop user if exists' on MySQL 5.5

### Steps to reproduce the behaviour
sudo docker run --name cairis-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -d mysql:5.5

### Version of CAIRIS
1.5.4

### Operating Systems / Hardware
Using images from dockerhub

### Remediation 
sudo docker run --name cairis-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -d mysql:5.5
Becomes
sudo docker run --name cairis-mysql -e MYSQL_ROOT_PASSWORD=my-secret-pw -d mysql:5.7

**Comment by failys** [comment]
Thanks for spotting this.  I've just fixed the install.md file that generates https://cairis.org/install , so the installation instructions should now be up-to-date.

I've just checked the script to build the live demo on https://demo.cairis.org (https://github.com/failys/cairis/blob/master/docker/buildDemo.sh) and this already uses the mysql:5.7 container, so it looks like the getting started guide was just a little out-of-date.
